.travis.yml

```diff
@@ -1,9 +1,6 @@
-
 language: go
-go: 
- - tip
-
+go:
+ - 1.6.3
```

Comment:
I see, I did not realize go was built for executing against tip, that seems excessive for less value. So in this case I'm happy to just test against 1.6.3.